pkg_resources/extern/__init__.py

```diff
@@ -20,17 +21,18 @@ def search_path(self):
         yield self.vendor_pkg + '.'
         yield ''
 
+    def _module_matches_namespace(self, fullname):
+        """Figure out if the target module is vendored."""
+        root, base, target = fullname.partition(self.root_name + '.')
+        return not root and any(map(target.startswith, self.vendored_names))
+
     def find_module(self, fullname, path=None):
         """
         Return self when fullname starts with root_name and the
         target module is one vendored through this importer.
```

Comment:
This comment no longer matches the implementation. Is `find_module` needed at all?

Reply:
Not sure, needs checking under Python 3.6. I don't fully understand the extent of the correctness of my change honestly. Not everything is well-documented so I tried "comparing notes" by reading the stdlib source but I'm still feeling lost. I think it still has `find_module()` but based on https://github.com/python/cpython/commit/57c6cb5100d19a0e0218c77d887c3c239c9ce435 you're probably right and it could be deleted (and maybe some other methods too).